### ISSUE TITLE
[BITV] 9.3.3.2/8.2 - The "Add group" input field does not have a visual label, it only has a placeholder element. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37090]
see also https://github.com/nextcloud/server/issues/37091

- [x] https://github.com/nextcloud-libraries/nextcloud-vue/pull/4494
- [x] Release
- [x] https://github.com/nextcloud/server/pull/40692
- [x] https://github.com/nextcloud/server/pull/40110

<img width="291" alt="61c68544e79ec5b016e35a21f21e15a7" src="https://user-images.githubusercontent.com/1315170/223564162-159c6335-14e1-4647-8448-84ea8c396c09.png">

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-9f92015c98-v8-n2

</details>

**Comment by Pytal** [comment]
Would need a redesign of NcAppNavigationNewItem

![image](https://github.com/nextcloud/server/assets/24800714/0d58bff7-3a1e-4496-8037-91fc6393dbe0)

Could be NcActions with NcActionInput inside and a visible label @nimishavijay?

**Comment by JuliaKirschenheuter** [comment]
@marcoambrosini?

**Comment by marcoambrosini** [comment]
Agree with @Pytal, we could use `NcAppnavigationCaption` which comes with an `NcActionMenu` on the right. 
This is exactly what we do for contacts groups.


<img width="391" alt="Screenshot 2023-08-29 at 10 49 50" src="https://github.com/nextcloud/server/assets/26852655/b2206e1e-55f8-418a-98eb-5c27b06dc99b">


And we should probably deprecate the NcAppNavigationNew component. cc @skjnldsv what are your thoughts on that?

**Comment by skjnldsv** [comment]
> cc @skjnldsv what are your thoughts on that?

None :)
As long as the component code and logic make sense, it doesn't trubles me. 
Nonetheless, using an `NcAppnavigationCaption` seems counter intuitive. SO maybe there can just be a bit of change in the `NcAppNavigationNew` to have it use the `NcAppnavigationCaption` :shrug:

**Comment by Pytal** [comment]
> None :) As long as the component code and logic make sense, it doesn't trubles me. Nonetheless, using an `NcAppnavigationCaption` seems counter intuitive. SO maybe there can just be a bit of change in the `NcAppNavigationNew` to have it use the `NcAppnavigationCaption` 🤷

Probably for a separate issue @skjnldsv ?

**Comment by Pytal** [comment]
While working on https://github.com/nextcloud/server/pull/40110 which uses NcAppNavigationCaption with actions I found a regression from https://github.com/nextcloud/server/pull/40576 which removes pointer events making the actions unclickable and changes the styling

![image](https://github.com/nextcloud/server/assets/24800714/544ad427-3bd7-42cc-8b93-f267e644fc78)

Can you scope the `.app-navigation-caption` styles to non-Vue app navigation containers again @JuliaKirschenheuter ?

**Comment by Pytal** [comment]
👆 fixed in https://github.com/nextcloud/server/pull/40858
